lib/gui/css/main.css

```diff
@@ -6402,30 +6402,30 @@ body {
 .component-drive-selector-body .list-group-item-text {
   word-break: break-all; }
 
-.modal-drive-selector-modal .list-group {
+.component-drive-selector-body .list-group {
```

Comment:
It looks like this might be related to #1073 ? But that PR _does_ also modify `lib/gui/css/main.css`, which is weird.

Observe:
```bash
andrew@shyknee:~/github/resin-io/etcher-clean$ git checkout 05fdd5c
Previous HEAD position was 70c7ddc... fix(GUI): don't auto select system drives in unsafe mode
HEAD is now at 05fdd5c... refactor(GUI): unify modal body styles (#1073)
andrew@shyknee:~/github/resin-io/etcher-clean$ git status
HEAD detached at 05fdd5c
nothing to commit, working directory clean
andrew@shyknee:~/github/resin-io/etcher-clean$ npm run sass

> etcher@1.0.0-beta.18 sass /home/andrew/github/resin-io/etcher-clean
> node-sass ./lib/gui/scss/main.scss > ./lib/gui/css/main.css

andrew@shyknee:~/github/resin-io/etcher-clean$ git status
HEAD detached at 05fdd5c
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   lib/gui/css/main.css

no changes added to commit (use "git add" and/or "git commit -a")
```

Is it possible that @jviotti somehow has a different version of `sass` installed, and that's why #1073 didn't "properly" update the CSS? :-/